Sources/Logging/Locks.swift

```diff
@@ -53,8 +54,12 @@ internal final class Lock {
         #if os(Windows)
         InitializeSRWLock(self.mutex)
         #else
-        let err = pthread_mutex_init(self.mutex, nil)
-        precondition(err == 0)
+        var attr = pthread_mutexattr_t()
+        pthread_mutexattr_init(&attr);
```

Comment:
formatter won't like those `;` :-)

@gwynne can you please run `./scripts/sanity.sh` to reformat?
(CI failure because of that)

Reply:
Whoops! Apparently I copy-pasted bad formatting that's present in NIO 😅

Reply:
😄 Thanks for fixing